proof/access-control/CNode_AC.thy

```diff
@@ -411,6 +392,29 @@ lemma dmo_wp:
   shows "\<lbrace>\<lambda>s. P s (machine_state s)\<rbrace> do_machine_op mop \<lbrace>R\<rbrace>"
   by (wpsimp simp: do_machine_op_def use_valid[OF _ mopv])
 
+(* FIXME: MOVE *)
+(* do_machine_op distributing over binds/basic operations*)
+lemma dmo_distr:
```

Comment:
Also move internally here, but would make sense to put somewhere high up in AInvs if we're touching it already.